src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -1286,6 +1286,14 @@ private WebElement getMsqMinSelectableChoicesBox(int qnNumber) {
         return browser.driver.findElement(By.id(getMsqMinSelectableChoicesBoxId(qnNumber)));
     }
 
+    public WebElement getMsqMaxSelectableChoicesBoxElement(int qnNumber) {
```

Comment:
What's the difference between this method and `getMsqMaxSelectableChoicesBox`

Reply:
I did look for existing method but the old method was private. Should have modified the old method instead?

Reply:
I think there's no difference with making a new `public` method that does the same thing, as making a `private` method `public`.

Reply:
My bad. Will change the private to public instead.